src/modify.js

```diff
@@ -88,4 +69,45 @@ exports.modifyMultipleServices = function (env, newImports) {
       resolve(resultPromise);
     });
   });
+}
+
+/*
+ * Services
+ */
+exports.modifyMultipleServices = function (env, newImports) {
+  return modifyLock(env, (json) => {
+    const imports = getMapFromManifest(json);
+    Object.assign(imports, newImports);
```

Comment:
It is now possible for `newImports` to be undefined, since you can call PATCH /import-map.json without providing the `imports` property.

`Object.assign(imports, undefined)` does not throw any errors or cause unexpected behavior. However, it unnecessarily seizes the modify lock and writes the object without the object having changed. Additionally, leaving the code like this makes it more likely for us to introduce a bug later because the code seems to assume that newImports is defined. Instead, we should change this to be similar to what modifyMultipleScopes is:

```js
 exports.modifyMultipleServices = function (env, newImports) {
  if (newImports) {
    return modifyLock(env, json => {
      const imports = getMapFromManifest(json);
      Object.assign(imports, newImports)
      return json
    }) 
  } else {
    return Promise.resolve()
  }
}
```